Xamarin/MvvmSeed/MvvmSeed/MvvmSeed.MigrationsTool/Program.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+
+namespace MvvmSeed.MigrationsTool
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            //NOTES: We need this console projects to generate migrations since EF Tools are not available for the other platforms yet
+            //TODO: Do something cool here to generate migrations automatically
+            Console.WriteLine("WIP");
```

Comment:
So creating migrations is possible on other platforms but the only issue is that the process is manual?